Makefile

```diff
@@ -25,22 +17,76 @@ ifeq ($(IMG), )
 endif
 # BUNDLE_IMG defines the image:tag used for the bundle.
 # You can use it as an arg. (E.g make bundle-build BUNDLE_IMG=<some-registry>/<project-name-bundle>:<tag>)
-ifeq ($(BUNDLE_IMG), )
-	BUNDLE_IMG = $(IMAGE_TAG_BASE)-bundle:v$(VERSION)
+BUNDLE_IMG ?= $(IMAGE_TAG_BASE)-bundle:v$(VERSION)
+
+# default platform type
+ODH_PLATFORM_TYPE ?= OpenDataHub
```

Comment:
we need somewhere in the README to tell how to use this env varable to build rhoai?